src/pages/api/index.ts

```diff
@@ -30,3 +31,10 @@ export const login = (loginUser: LoginUser) =>
 
 export const signUp = (newUser: NewUser) =>
   axios.post(`${process.env.NEXT_PUBLIC_API_URL}/signup`, newUser).then(({ data }) => data);
+
+export const logout = async (token: string) => {
+  const headers = {
+    Authorization: `Bearer ${token}`,
```

Comment:
might wanna create a reusable function for this since you're gonna be doing this header thing a lot